metadata/recreation/boat_ramps.md

```diff
@@ -0,0 +1,73 @@
+# Title
+
+Utah Boat Ramps
+
+## ID
+
+82f7dae2-1d73-4a6c-836a-7cc0619d048e
+
+## Brief Summary
+
+Boat ramps and their associated water body, administrator, and allowed vessel types.
+
+## Summary
+
+This dataset contains points representing approximate locations for boat ramps found in Utah. Features include the body of water associated with the ramp, administrators, and allowed vessel types. This dataset does not contain information on permits, pricing, watercraft inspections, or hours of operation.
+
+## Description
+
+### What is the dataset?
+
+Boat ramps are sloping surfaces used to launch small and retrieve small watercraft from a body of water. This dataset contains approximate locations for boat ramps found in Utah.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general cartographic and reference purposes.
+
+### What does the dataset represent?
+
+Each point in this dataset indicates the approximate location of a boat ramp in Utah. Features include the ramp name, associated body of water, whether the ramp is state, federal, or private (owner), the managing agency, the administrator, and allowed vessel types.
+
+### How was the dataset created?
+
+UGRC digitized this layer using aerial imagery and assigned attributes using data submitted by government partners.
+
+<!--- A guesstimate. Do we have info on this dataset was created or how it's being maintained? --->
+
+### How reliable and accurate is the dataset?
+
+<!--- Do we have a way of knowing if these data are comprehensive? --->
+
+## Credits
```

Comment:
We can update the credits in the future if we get help updating the data (like by Utah State Parks or Outdoor Recreation).